static/sass/_pattern_subscriptions.scss

```diff
@@ -84,12 +84,49 @@
     padding: $card-padding;
 
     @media only screen and (max-width: $breakpoint-small) {
+      // On small screens the details are hidden when not active.
       display: none;
 
+      // Show the modal when the details are active.
       &.is-active {
         display: flex;
       }
     }
+
+    // Remove modal classes on large screens.
```

Comment:
@anthonydillon because you can't [extend inside a media query](https://sass-lang.com/documentation/at-rules/extend#extend-in-media.) I ended up doing the inverse and removed the classes on large screens. Can you think of a better way of doing this?